funnel/registry.py

```diff
@@ -197,7 +204,8 @@ class LoginProvider:
     icon = None
     #: Login form, if required
     form = None
-    #: This service's usernames are typically used for addressing with @username
+    #: This service's usernames are typically
+    # used for addressing with @username
```

Comment:
This line needs to use `#:`. The colon is used by Sphinx for documentation of whatever is on the following line, since objects in Python can't have docstrings, only modules, classes and files.

In general, wrap comments as close to 88th column as possible. When we find a comment reformatter, it'll use a similar rule.